scripts/create-client-mock-data.js

```diff
@@ -13,6 +17,24 @@ async function main() {
 
   const mockData = JSON.parse(data)
 
+  mockData.footprint.forEach((footprint) => {
+    let updatedMonthStr = updatedMonth.toISOString()
+    footprint.timestamp = updatedMonthStr
```

Comment:
You can avoid the need for a variable by assigning inline 
```suggestion
    footprint.timestamp = updatedMonth.toISOString()
```